molecule/driver/docker.py

```diff
@@ -119,6 +119,11 @@ class Docker(base.Base):
               less secure. For details, please reference `Docker Security
               Configuration`_
 
+    .. note:: With the environment variable ``DOCKER_HOST`` the user can define
+              which `Docker`_ daemon to use. Do note that the `Docker`_ binary
+              must be installed on the machine that run Molecule even if the 
```

Comment:
```suggestion
              must be installed on the machine that runs Molecule even if the 
```

Reply:
It would worth mentioning that user can even use ssh transport protocol to connect to remote docker, something that works well for both docker-cli and docker-py but is not well documented, one of the reasons why I wrote https://docs.sbarnea.com/molecule/using-a-remote-docker some time ago.

The statement is false: user does *not* need to have docker installed on the localhost when using remoting. Docker-py talks directly with the remote host and you can avoid having to install docker locally. Molecule is using only docker-py (via ansible modules)